open_lm/datapreprocess/ray/tokenize_shuffle.py

```diff
@@ -466,12 +540,12 @@ def main(args):
     parser.add_argument("--subset", type=int, default=None)
     parser.add_argument("--subfraction", type=float, default=None)
     parser.add_argument("--ray_address", type=str, default=None)
-    parser.add_argument("--block_size", type=str, default="10MB")
+    parser.add_argument("--num_writers_per_node", type=int, default=1)
```

Comment:
Do we have to set our writers on a per node basis?

Reply:
yes.

Reply:
Got it, thanks!